web3.js/test/vote-program.test.ts

```diff
@@ -154,13 +173,57 @@ describe('VoteProgram', () => {
         lamports: LAMPORTS_PER_SOL,
         toPubkey: recipient.publicKey,
       });
-
       await sendAndConfirmTransaction(connection, withdraw, [authorized], {
         preflightCommitment: 'confirmed',
       });
       expect(await connection.getBalance(recipient.publicKey)).to.eq(
         LAMPORTS_PER_SOL,
       );
+
+      const newAuthorized = Keypair.generate();
+      await helpers.airdrop({
+        connection,
+        address: newAuthorized.publicKey,
+        amount: LAMPORTS_PER_SOL,
+      });
+      expect(await connection.getBalance(newAuthorized.publicKey)).to.eq(
+        LAMPORTS_PER_SOL,
+      );
+
+      // Authorize a new Voter.
+      let authorize = VoteProgram.authorize({
+        votePubkey: newVoteAccount.publicKey,
+        authorizedPubkey: authorized.publicKey,
+        newAuthorizedPubkey: newAuthorized.publicKey,
+        voteAuthorizationType: VoteAuthorizationLayout.Voter,
+      });
+      await sendAndConfirmTransaction(connection, authorize, [authorized], {
+        preflightCommitment: 'confirmed',
+      });
+
+      // Authorize a new Withdrawer.
+      authorize = VoteProgram.authorize({
+        votePubkey: newVoteAccount.publicKey,
+        authorizedPubkey: authorized.publicKey,
+        newAuthorizedPubkey: newAuthorized.publicKey,
+        voteAuthorizationType: VoteAuthorizationLayout.Withdrawer,
+      });
+      await sendAndConfirmTransaction(connection, authorize, [authorized], {
+        preflightCommitment: 'confirmed',
+      });
+
+      // Test old authorized cannot witdraw anymore.
+      withdraw = VoteProgram.withdraw({
+        votePubkey: newVoteAccount.publicKey,
+        authorizedWithdrawerPubkey: authorized.publicKey,
+        lamports: minimumAmount,
+        toPubkey: recipient.publicKey,
+      });
+      await expect(
+        sendAndConfirmTransaction(connection, withdraw, [authorized], {
+          preflightCommitment: 'confirmed',
+        }),
+      ).to.be.rejected;
```

Comment:
Do you mind adding one more transaction to demonstrate a successful withdraw with the new authority?